### DISCUSSION TITLE
Detached a Paste Sketch symbol? Let us know why!

### DISCUSSION THREAD

**Discussion by serifluous** [twilio-labs/paste#416] [General] [answered]
Ever needed to detach a Sketch symbol in the Paste library? Let us know which one, so we can make our library a better out-of-the-box experience for you!

Let us know:

**Component and variant:** _e.g., default modal with left action_

**Theme:** _e.g., Console_

**Why you detached and what you were trying to achieve:** _e.g., I wanted to put an icon and some text in the footer, but I couldn't override any of the symbols in the footer; I needed to figure out what the spacing in a modal should be._

**Link to your artboard:**

**Comment by sangster009** [comment, marked as answer]
I need to design a table with the Select components in rows. Right now, I can't remove the label from the component.

**Reply by serifluous** [reply]
Thanks for the context! If you need just the Select field, detaching is actually ok here so that you can get the right bounding box for the symbol.

Please remember to:
- Have a clear table header that indicates what users will be selecting in the Select fields. The table header should basically say what the Select labels should say.
- Use the [Screen Reader Only component](https://paste.twilio.design/components/screen-reader-only) to communicate labels on each Select. It'll help give context to assistive technology.

@SiTaggart chime in if there's anything else to add/clarify ^
